lib/resty/aes.lua

```diff
@@ -125,50 +125,75 @@ cipher = function (size, _cipher)
 end
 _M.cipher = cipher
 
-function _M.new(self, key, salt, _cipher, _hash, hash_rounds)
+
+local options = {
+    default = 0x01,
+    key_without_gen = 0x02
```

Comment:
Better add a trailing comma to simplify adding more options in the future.

Reply:
Also, the option name"key_without_gen" is confusing. What does it mean?

BTW, shall we add some docs to README for this change or feature?

Reply:
**about "key_without_gen" option**
By default, new() function in aes.lua uses EVP_BytesToKey in openssl to generate key and iv from input key together with salt and etc by md5. Use generated key can avoid some attacks. 
But sometimes people just want to use simple key without iv and generation instead, for compatibility or other reason.
so how should we name it?  use_origin_key?

**about docs** 
Of course we should. It takes me quite a while when I crash into this.

Reply:
@haorenfsa `use_raw_key` might be a better name? Also, maye we do not need the `default` option at all?

Reply:
yeah，it's better. And default can be replaced with nil input